ui/packages/consul-ui/app/components/consul/auth-method/search-bar/index.hbs

```diff
@@ -0,0 +1,82 @@
+<form
+  class="consul-auth-method-search-bar filter-bar"
+  ...attributes
+>
+  <div class="search">
+    <FreetextFilter
+      @onsearch={{action @onsearch}}
+      @value={{@search}}
+      @placeholder="Search"
+    >
+      <PopoverSelect
+        class="type-search-properties"
+        @position="right"
+        @onchange={{action @onfilter.searchproperty}}
+        @multiple={{true}}
+      as |components|>
+        <BlockSlot @name="selected">
+          <span>
+            Search across
+          </span>
+        </BlockSlot>
+        <BlockSlot @name="options">
+  {{#let components.Optgroup components.Option as |Optgroup Option|}}
+          <Option @value="Name" @selected={{contains 'Name' @filter.searchproperties}}>Name</Option>
+          <Option @value="DisplayName" @selected={{contains 'DisplayName' @filter.searchproperties}}>Display Name</Option>
+  {{/let}}
+        </BlockSlot>
+      </PopoverSelect>
+    </FreetextFilter>
+  </div>
+  <div class="filters">
+    <PopoverSelect
+      class="select-type"
+      @position="left"
+      @onchange={{action @onfilter.type}}
+      @multiple={{true}}
+    as |components|>
+      <BlockSlot @name="selected">
+        <span>
+          Type
+        </span>
+      </BlockSlot>
+      <BlockSlot @name="options">
+{{#let components.Optgroup components.Option as |Optgroup Option|}}
+        <Option class="kubernetes" @value="kubernetes" @selected={{contains 'kubernetes' @filter.types}}>Kubernetes</Option>
+        <Option class="jwt" @value="jwt" @selected={{contains 'jwt' @filter.types}}>JWT</Option>
+        <Option class="oidc" @value="oidc" @selected={{contains 'oidc' @filter.types}}>OIDC</Option>
+{{/let}}
```

Comment:
OIDC is only relevant when SSO is enabled (i.e. in Ent), we should probably wrap a conditional around this so its only visible when SSO is enabled.